packages/compass-indexes/src/components/search-indexes-table/search-indexes-table.tsx

```diff
@@ -73,7 +77,28 @@ export const SearchIndexesTable: React.FunctionComponent<
   onSortTable,
   openCreateModal,
   status,
+  onDropIndex: onConfirmDropIndex,
 }) => {
+  const onDropIndex = useCallback(
+    async (name: string) => {
+      const isConfirmed = await showConfirmation({
+        title: `Are you sure you want to drop "${name}" from Cluster?`,
+        buttonText: 'Drop Index',
+        variant: 'danger',
+        requiredInputText: name,
+        description: (
+          <Body>
+            If you drop default, all queries using it will no longer function
+          </Body>
+        ),
+      });
+      if (!isConfirmed) {
+        return;
+      }
```

Comment:
Because this is part of the business logic, deleting should work consistently from all the places where it can be called, and for consistency with other places where we use this method, this should probably be part of the action and not UI

Reply:
Moved it [518c3a1](https://github.com/mongodb-js/compass/pull/4844/commits/518c3a15dd40b9a7bcdc727d3d6afdde095cd902)